src/main.cpp

```diff
@@ -422,6 +391,7 @@ struct CNodeState {
         nStallingSince = 0;
         nDownloadingSince = 0;
         nBlocksInFlight = 0;
+        nHeadersInFlight = 0;
```

Comment:
it doesn't seem like this parameter is ever used?

Reply:
yes, its left from some code idea i was sketching. ill remove it, good eye